src/components/structures/FilePanel.tsx

```diff
@@ -28,25 +31,33 @@ import {RightPanelPhases} from "../../stores/RightPanelStorePhases";
 import DesktopBuildsNotice, {WarningKind} from "../views/elements/DesktopBuildsNotice";
 import {replaceableComponent} from "../../utils/replaceableComponent";
```

Comment:
```suggestion
import DesktopBuildsNotice, { WarningKind } from "../views/elements/DesktopBuildsNotice";
import { replaceableComponent } from "../../utils/replaceableComponent";
```

and RightPanelPhases above but it won't let me comment on it :(